### PR TITLE
Fix issue with Multicompany

### PR DESCRIPTION
**Pull request by mapiolca**
Before : Show all letters of all entity in contact tab

After : show only current entity letters in contact tab

### CODE REVIEW

**Comment by atm-john** [comment]
Have you seen my comment on https://github.com/ATM-Consulting/dolibarr_module_externalaccess/pull/16 ?

**Comment by mapiolca** [comment]
No I haven't.

File: class/referenceletterselements.class.php [hunk 1]
```diff
@@ -367,7 +368,7 @@ public function fetchAll($sortorder = '', $sortfield = '', $limit = 0, $offset =
 		$sql .= " ,p.title as title_referenceletters";
 		$sql .= " FROM " . MAIN_DB_PREFIX . "referenceletters_elements as t";
 		$sql .= " INNER JOIN " . MAIN_DB_PREFIX . "referenceletters as p ON p.rowid=t.fk_referenceletters";
-		$sql .= " WHERE t.entity IN (" . getEntity('referenceletters') . ")";
+		$sql .= " WHERE t.entity IN (" . getEntity('referenceletter') . ")";
```
Comment (atm-john) [inline class/referenceletterselements.class.php]: referenceletters or referenceletter ?

Comment (mapiolca) [reply class/referenceletterselements.class.php]: referenceletters

**Pull request merged**